utils/frame/try-runtime/cli/src/commands/on_runtime_upgrade.rs

```diff
@@ -29,12 +30,16 @@ pub struct OnRuntimeUpgradeCmd {
 	#[command(subcommand)]
 	pub state: State,
 
-	/// Execute `try_state`, `pre_upgrade` and `post_upgrade` checks as well.
+	/// Select which optional checks to perform:
 	///
-	/// This will perform more checks, but it will also makes the reported PoV/Weight be
-	/// inaccurate.
-	#[clap(long)]
-	pub checks: bool,
+	/// - `all`: Perform all checks (default).
+	/// - `pre-and-post`: Perform pre- and post-upgrade checks.
+	/// - `try-state`: Perform the try-state checks.
+	/// - `none`: Perform no checks.
+	///
+	/// Performing any checks will potentially invalidate the measured PoV/Weight.
+	#[clap(long, default_value = "All", verbatim_doc_comment)]
```

Comment:
I think in general it would be better to use `enum_arg` for clap. However, this would require to duplicate the enum which isn't that nice.

Reply:
Yea it looks slightly nicer now, since instead of the manual `FromStr` i just need a conversion and can use the Clap parsing and CLI help output.